### DISCUSSION TITLE
Java Installed Adapter deprecation

### DISCUSSION THREAD

**Discussion by daniel-shuy** [keycloak/keycloak#30326] [Adapter deprecation]
I understand that the Java adapters have been deprecated and removed because most common Java frameworks (Spring, Jakarta EE, etc) already support OpenID Connect.

But there doesn't seem to be any alternative for the Java client applications that were using the [keycloak-installed-adapter](https://www.keycloak.org/docs/24.0.5/securing_apps/#_installed_adapter).

Maybe the `keycloak-installed-adapter` should continue to be maintained, like the JavaScript adapter (`keycloak.js`).

**Comment by sennew** [comment]
Is there any information on a replacement of the keycloak-installed-adapter for java client desktop applications, like java swing applications?
